App.tsx

```diff
@@ -1,18 +1,231 @@
-import { StyleSheet, Text, View } from 'react-native';
+import { useState } from "react";
+import { Pressable, StyleSheet, Text, View } from "react-native";
 
 export default function App() {
+  const [display, setDisplay] = useState("0");
+  const [firstInput, setFirstInput] = useState<string | undefined>();
+  const [secondInput, setSecondInput] = useState<string | undefined>();
+  const [operator, setOperator] = useState<string | undefined>();
+
+  const clearAll = () => {
+    resetInputs();
+    setDisplay("0");
+  };
+
+  const resetInputs = () => {
+    setFirstInput(undefined);
+    setSecondInput(undefined);
+    setOperator(undefined);
+  };
+
+  const calculateResult = () => {
+    if (operator === "/" && secondInput === "0") {
+      // Set error on divide by zero
+      setDisplay("Error");
+      return;
+    }
+
+    // TODO: Add validation
```

Comment:
I'll remove this `TODO` before merging. I'm a strong proponent of creating backlog tickets for TODOs instead of leaving them in the code where they before "overgrow" and lose context over time. In this case, I forgot to remove it before it's not actually necessary, but it seemed like a good callout for this PR self-review 😄

Reply:
`TODO` removed (for demo purposes) [2fdeab0](https://github.com/grantbartone/RN_Calculator/pull/1/commits/2fdeab0e58e7111ee3678b5c87386ff4ffe71e43)